src/components/AnchoredTextLabels.h

```diff
@@ -32,9 +37,13 @@ namespace REGoth
       bs::GUILabel* mLabel;
     };
 
+    bool isPointWithinForwardDrawDistance(const bs::Transform& cameraTransform, const bs::Vector3& point);
```

Comment:
That can be `const`. Otherwise it looks fine!

Reply:
Good point. Done in 72fbf32.